src/dashboard-refactor/types.tsx

```diff
@@ -1,5 +1,24 @@
+export interface NewItemsCountState {
+    displayNewItemsCount: boolean
+    newItemsCount: number
+}
+
+export interface DroppableState {
+    isDroppable: boolean // this defines whether items can be dropped (not whether there is a state change on drag-over)
+    isDraggedOver: boolean
+    isBlinking: boolean
```

Comment:
`DroppableState` sounds like all things related to drag and drop. `isBlinking` is something specfifc to some component using this, so it should be there instead.

Reply:
When isBlinking = true it triggers the animation to signal that that object has received the dropped item. Feels like it's inherently part of the 'DroppableState' (which I've now renamed to 'DropReceivingState'). Perhaps it should be a timestamp for the most recent drop receipt instead.

Reply:
Hmmm no that makes the render implementation too complicated. I've settled for triggerSuccessfulDropAnimation: boolean. The onDrop handler will just have to set a timeout to change it back.

Alles gut?

Reply:
Waiting on confirmation of this approach before I make a pull request for all of the PR feedback